app/services/data_migrations/correct_hesa_ethnicity.rb

```diff
@@ -0,0 +1,21 @@
+module DataMigrations
+  class CorrectHesaEthnicity
+    TIMESTAMP = 20241024110710
+    MANUAL_RUN = false
+
+    def change
+      application_forms.update_all(
```

Comment:
Is it worth doing this in batches? Not sure how many records we're updating here so might not be needed

Reply:
Just 5k so I think it's not necessary